functional-tests/fn_client_restart.py

```diff
@@ -51,7 +51,9 @@ def main(self, ctx: flexitest.RunContext):
 def check_already_sent_proof(seqrpc):
     try:
         # Proof for checkpoint 0 is already sent
-        seqrpc.strataadmin_submitCheckpointProof(0, "abc123")
+        seqrpc.strataadmin_submitCheckpointProof(
+            0, "000000000000000000"
```

Comment:
So, `000000000000000000` is the default proof now ?
Earlier it was empty string.

Reply:
Yes. Earlier `Proof` only included the raw proof. Now it also includes the id.